v3.3/glfw/glfw_tree_rebuild.go

```diff
@@ -0,0 +1,10 @@
+package glfw
+
+//go:generate ../../scripts/glfw_tree_rebuild.sh
+
+// This is a recursive hash of the full contents of the upstream glfw, as
```

Comment:
nits: `upstreamTreeSHA is ...` for Go's comment convention?

Reply:
Ah, my go idioms are a little rusty. Fixed 211ecac01a39078a11f649cfd964e30d2ca90bd9. Thanks ;-)